plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererFeatureConfigService.kt

```diff
@@ -80,17 +80,20 @@ class CodeWhispererFeatureConfigService {
     // 6) Add a test case for this feature.
     fun getTestFeature(): String = getFeatureValueForKey(TEST_FEATURE_NAME).stringValue()
 
+    fun getIsDataCollectionEnabled(): Boolean = getFeatureValueForKey(DATA_COLLECTION_FEATURE, false).boolValue()
+
     fun getCustomizationArnOverride(): String = getFeatureValueForKey(CUSTOMIZATION_ARN_OVERRIDE_NAME).stringValue()
 
     // Get the feature value for the given key.
     // In case of a misconfiguration, it will return a default feature value of Boolean true.
-    private fun getFeatureValueForKey(name: String): FeatureValue =
+    private fun getFeatureValueForKey(name: String, defaultValue: Boolean = true): FeatureValue =
```

Comment:
better default to false

Reply:
it defaults to false for this one, I'm not sure if there is a reason other features defaults to true so not touching that

Reply:
actually this is not needed since it's a string value